rfc/rfc-60/rfc-60.md

```diff
@@ -97,22 +97,21 @@ public interface HoodieStorageStrategy extends Serializable {
 }
 ```
 
-### Generating file paths for object store optimized layout
+### Generating File Paths for Object Store Optimized Layout
 
 We want to distribute files evenly across multiple random prefixes, instead of following the traditional Hive storage
 layout of keeping them under a common table path/prefix. In addition to the `Table Path`, for this new layout user will
 configure another `Table Storage Path` under which the actual data files will be distributed. The original `Table Path` will
```

Comment:
do we see a need to support multiple storage paths, to distribute data across buckets?

Reply:
Yes, users may want to have multiple buckets and have different access controls/encryptions on different buckets. It's not a common use case but it would be nice to enable that